sympy/solvers/tests/test_inequalities.py

```diff
@@ -318,6 +318,13 @@ def test_issue_8974():
     assert isolve(oo > x, x) == And(-oo < x, x < oo)
 
 
+def test_issue_10198():
+    assert reduce_inequalities(-1 + 1/abs( 1/x - 1 ) < 0) == Or(And(-oo < x, x < 0), And(S(0) < x, x < S(1)/2))
```

Comment:
- [ ] no space after or before partheses
- [ ] space after commas but not before
- [ ] line length less than 72; for something like line 325 where the expression is inside parentheses, you can simply break before the last Abs, lining it up with the Or at the start of the line
- [ ] be careful to not introduce new whitespace errors (`bin/test quality` is your friend)